ast/env.go

```diff
@@ -342,55 +348,70 @@ func (n *typeTreeNode) Insert(path Ref, tpe types.Type) {
 		leafs := curr.Leafs()
 		for p, t := range leafs {
 			var err error
-			curr.value, err = insertIntoObject(curr.value.(*types.Object), *p, t)
+			curr.value, err = insertIntoObject(curr.value.(*types.Object), *p, t, env)
 			if err != nil {
 				panic(fmt.Errorf("unreachable, insertIntoObject: %w", err))
 			}
 		}
 	}
 }
 
-func insertIntoObject(o *types.Object, path Ref, tpe types.Type) (*types.Object, error) {
-	if len(path) == 0 {
-		return o, nil
+func (n *typeTreeNode) String() string {
```

Comment:
Was this added for debugging purposes or does a user ever see that output? (Because then we'd need tests, I suppose.)

Reply:
Yes, this was for debugging purposes. It was very useful, so I decided to leave it in. There is no active code path linking here, but Intellij can make use of it when debugging.

Reply:
I like these things, let's keep it.